pages.fr/common/borg.md

```diff
@@ -0,0 +1,33 @@
+# borg
+
+> Outil de sauvegarde avec déduplication.
+> Crée des sauvegarde distante ou locale qui peuvent être montée comme un système de fichier.
```

Comment:
```suggestion
> Crée des sauvegardes distantes ou locales qui peuvent être montées comme un système de fichiers.
```

Also I am not sure if `des` could be omitted. We tend to say "create backups" instead of "create *the* backups" in English. Would that make sense in French? I'm curious.

Reply:
no we cannot remove `des`.
anyway thanks for your review, I'm quite ashamed of errors I've made in my mother tongue.